modules/desktop/graphics/hardware.nix

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{lib, ...}:
+with lib; {
+  options.ghaf.graphics.hardware = {
+    networkDevice = mkOption {
+      type = types.anything;
+      default = {};
+      description = ''
+        Network device interface for use with graphics stack.
```

Comment:
Maybe add and `example = {}` value for an interface. From the description it's not clear what needs to be set here. Without reading the description, I would expect to set the string of the network interface name.

Reply:
It is the same definition as in hardware namespace - copied that over including example so that it is clear. Ideally, we can remove this from here in the future.